Makefile

```diff
@@ -90,8 +90,19 @@ feature_manifests:
 		curl $$url -o $$file; \
 	done
 
-
-fetch_external_resources: jetstream_config feature_manifests
+convert_feature_manifests:
+	cd experimenter/experimenter/features/manifests/;\
+	curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh -s -- -y;\
+	source "$HOME/.cargo/env";\
+	git clone git@github.com:mozilla/application-services.git;\
+	cd application-services;\
+	git submodule init;\
+	git submodule update --recursive;\
+	cd components/support/nimbus-fml;\
+	cargo build;\
+	cargo run "../../../../monitor-web.fml.yaml" experimenter --output "../../../../monitor-web.yaml" --channel developer;
```

Comment:
You can actually eliminate the `cargo build` step if you want — `cargo run` does the build if it's not been built yet.

Also, you can get around needing to `cd` into `nimbus-fml` using `--manifest-path`
```
cargo run --manifest-path components/support/nimbus-fml/Cargo.toml <rest-of-command>
```
Just thought that might cut down on the `../`s needed

Reply:
Do we care if we do a release build or not?

Reply:
Not for this particular use case imo